samples/Uno.Toolkit.WinUI.Samples/Uno.Toolkit.WinUI.Samples.Windows.Package/Uno.Toolkit.WinUI.Samples.Windows.Package.wapproj

```diff
@@ -77,6 +77,7 @@
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22000.196">
       <IncludeAssets>build</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Uno.SourceGenerationTasks" Version="4.1.0" />
```

Comment:
Why is it needed in this project?

Reply:
I added those to get around the ReadOnlySpan error before I realized I just needed a newer dev version of Uno. Still need to go through and clean this all up